.gitignore

```diff
@@ -21,6 +21,7 @@ lib64/
 parts/
 sdist/
 var/
+data/
```

Comment:
let's not do git-lfs in a public repo as will likely be a big hurdle for novice contributors

Reply:
Agreed. I'll update, and zip the sample data and send to you.

Reply:
we can also do a separate optional gh repo, hotlink, or some other service folks use (s3, ...)